angular/projects/spark-angular/src/lib/components/inputs/Monetary.stories.ts

```diff
@@ -203,59 +220,66 @@ export const validating = () => ({
         Payment
       </label>
       <input
-        class="sprk-b-TextInput--has-text-icon"
-        [ngClass]="{'sprk-b-TextInput--error': monetaryInput.invalid}"
+        class="sprk-b-TextInput--has-text-icon sprk-b-TextInput--error"
+        aria-invalid="true"
         name="monetary_input"
         type="text"
-        pattern="^[+-]?[0-9]{1,3}(?:,?[0-9]{3})*(?:\.[0-9]{2})?$"
         [(ngModel)]="monetary_input"
         #monetaryInput="ngModel"
         sprkInput
       />
-      <div
-        [hidden]="monetaryInput.valid || monetaryInput.pristine"
-        sprkFieldError
-      >
+      <span sprkFieldError>
         <sprk-icon
-          iconType="exclamation-filled"
+          iconName="exclamation-filled"
           additionalClasses="sprk-b-ErrorIcon"
         ></sprk-icon>
-        <div class="sprk-b-ErrorText">Invalid amount.</div>
-      </div>
+        <div class="sprk-b-ErrorText">There is an error on this field.</div>
+      </span>
     </sprk-icon-input-container>
-  `
+  `,
 });
 
-export const formattingAndValidating = () => ({
+legacyInvalidMonetaryInput.story = {
+  name: 'Legacy Invalid (Deprecated)',
+  parameters: {
+    jest: [
+      'sprk-icon-input-container.component',
+      'sprk-input.directive',
+      'sprk-label.directive',
+      'sprk-field-error.directive',
+    ],
+  },
+};
+
+export const legacyDisabledMonetaryInput = () => ({
   moduleMetadata: modules,
   template: `
     <sprk-icon-input-container
       iconContainerClasses="sprk-b-TextInputIconContainer--has-text-icon"
     >
-      <label class="sprk-b-Label--monetary" sprkLabel>
+      <label class="sprk-b-Label--monetary sprk-b-Label--disabled" sprkLabel>
```

Comment:
We added the disabled class
here. But let's double check this. There might be an isDisabled input on sprkLabel we would use instead.

Reply:
Also for - id missing.

Reply:
I added the for/id. I think that because this is a legacy story is why it is using the class and not the input